### ISSUE TITLE
packet order

### ISSUE THREAD

**Issue by cithon** [vin350/AutoAttack#23]
flags kill aura: post while the player is moving;

post is just like bad packet order or similar;

probably some bug with moving and hitting packets. not sure how to fix; might have to use 1 tick delay

**Comment by Agentew04** [comment]
Something like #19 maybe?

**Comment by cithon** [comment]
yeah i think #19 is likely one of the only possible solutions but the optimal solution would hopefully have no delay and no bad packet order

**Comment by Agentew04** [comment]
I really don't know anything about packets and their ordering in Minecraft, so I will look into implementing the 1 tick delay probably this weekend
